src/AppInstallerCLICore/Argument.h

```diff
@@ -31,7 +32,7 @@ namespace AppInstaller::CLI
     };
 
     // Controls the visibility of the field.
-    enum class Visibility
+    enum class VisibilityArg

```

Comment:
>VisibilityArg [](start = 15, length = 13)

I would suggest moving it under Argument rather than just smooshing names together.